client-network/Cargo.toml

```diff
@@ -12,3 +12,5 @@ chain-tx-validation = { path = "../chain-tx-validation" }
 client-common = { path = "../client-common" }
 client-core = { path = "../client-core" }
 secstr = "0.3.2"
+secp256k1zkp = { git = "https://github.com/crypto-com/rust-secp256k1-zkp.git", rev = "a1e70e1f0bb65d3a97f100f79d99c85395d897f8", features = ["recovery", "endomorphism"] }
```

Comment:
I don't think we should add this dependency in here. Unless we explicitly need to use some of its functionality in this crate.

Reply:
agree